uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-widget/leave-widget-balance.component.spec.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/lodash',
+  'leave-absences/shared/components/leave-widget/leave-widget-balance.component',
+  'mocks/apis/entitlement-api-mock'
+], function (_) {
+  describe('LeaveWidgetBalance', function () {
+    var $componentController, $provide, $rootScope, $scope,
+      currentAbsencePeriod, absenceTypes, ctrl, Entitlement;
+    var contactId = 101;
+
+    beforeEach(module('leave-absences.components', 'leave-absences.mocks',
+    function (_$provide_) {
```

Comment:
I would indent `function (_$provide_) {` with 2 spaces, it was a bit difficult to read:
```js
beforeEach(module('leave-absences.components', 'leave-absences.mocks',
  function (_$provide_) {
    $provide = _$provide_;
  }));
```